src/metrics.jl

```diff
@@ -29,6 +29,40 @@ type CorrDist <: SemiMetric end
 type ChiSqDist <: SemiMetric end
 type KLDivergence <: PreMetric end
 
+"""
+    RenyiDivergence(α::Real)
+    renyi_divergence(P, Q, α::Real)
+
+Create a Rényi premetric of order α.
+
+Rényi defined a spectrum of divergence measures generalising the
+Kullback–Leibler divergence (see `KLDivergence`). The divergence is
+not a semimetric as it is not symmetric. It is parameterised by a
+parameter α, and is equal to Kullback–Leibler divergence at α = 1:
+
+At α = 0, ``R_0(P | Q) = -log(sum_{i: p_i > 0}(q_i))``
+
+At α = 1, ``R_1(P | Q) = sum_{i: p_i > 0}(p_i log(p_i / q_i))``
+
+At α = ∞, ``R_∞(P | Q) = log(sup_{i: p_i > 0}(p_i / q_i))``
```

Comment:
In math sections, Unicode literals should be replaced with the equivalent LaTeX, e.g. `\infty` in place of `∞`